resources/views/component/footer/footer.blade.php

```diff
@@ -0,0 +1,57 @@
+<footer class="bg-white dark:bg-gray-900">
+    <div class="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+        <div class="md:flex md:justify-between">
+          <div class="mb-6 md:mb-0">
+              <a href="https://flowbite.com/" class="flex items-center">
+                  {{-- Logo --}}
+                  <span class="self-center text-2xl font-semibold whitespace-nowrap dark:text-white italic">EnvioStore</span>
+              </a>
+          </div>
+          {{-- Content --}}
+          <div class="grid grid-cols-2 gap-8 sm:gap-6 sm:grid-cols-3">
+            {{-- Resources --}}
+              <div>
+                  <h2 class="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white">Resources</h2>
```

Comment:
mungkin pake text-lg ato text-md

Reply:
resolved

Reply:
ndak perlu tulis resolved the, nanti klo lu commit push, otomatis dibuat resolved